Week3/grocery.py

```diff
@@ -0,0 +1,15 @@
+grocery_list = {}
+while True:
+    try:
+        item = input()
+        if item in grocery_list:
+            grocery_list[item] += 1
+        else:
+            grocery_list[item] = 1
+
+    except EOFError:
```

Comment:
so if you press `return` key, the `input()` function will throw `EOFError` exception?  otherwise how the while loop is stopped?